ezCMakeConfig.cmake

```diff
@@ -19,3 +19,4 @@ set_property(GLOBAL PROPERTY EZ_CONFIG_QT_WINX64_URL "https://github.com/ezEngin
 set_property(GLOBAL PROPERTY EZ_CONFIG_VULKAN_SDK_LINUXX64_VERSION "1.3.275.0")
 set_property(GLOBAL PROPERTY EZ_CONFIG_VULKAN_SDK_LINUXX64_URL "https://sdk.lunarg.com/sdk/download/1.3.275.0/linux/vulkansdk-linux-x86_64-1.3.275.0.tar.xz")
 
+set(EZ_SUPPORT_TRACY ON CACHE BOOL "Support Tracy Profiler")
```

Comment:
Since tracy replaces the builtin profiling I don't think it should be on by default.